sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -313,6 +348,10 @@ static gnrc_rpl_parent_t *_gnrc_rpl_find_preferred_parent(gnrc_rpl_dodag_t *doda
                       (dodag->default_lifetime * dodag->lifetime_unit) * SEC_IN_MS);
     }
 
+#ifdef MODULE_GNRC_RPL_P2P
+    }
+#endif
+
     dodag->my_rank = dodag->instance->of->calc_rank(dodag->parents, 0);
```

Comment:
@BytesGalore I addressed this issue. Could you have a look at commit https://github.com/RIOT-OS/RIOT/pull/3721/commits/14fe84de7d72da0e0c051fa475fbf240b339a8d7 ?